### PR TITLE
Added orientationchange event, and destroy method

### PR DESCRIPTION
**Pull request by lisandropuzzolo**
In order to use this plugin in dinamic pages with templating engines as knockout, I added the destroy method.

I also added orientation change event to make it resize.

### CODE REVIEW

**Comment by mattnish** [comment]
Looks good, thanks for the pull request.  I'm going to test it out and will merge it after the weekend if everything runs smooth.

**Comment by lisandropuzzolo** [comment]
Great! Thank you for having this pull request in consideration.

File: rad.js [hunk 1]
```diff
@@ -18,9 +22,10 @@
         self.init();
         self.setScale();
 
-        $(window).resize(function(){
-            self.setScale();
-        });
+		self.__setScale = $.proxy( self.setScale, self );
+
+        $(window).on( 'resize.rad.' + this.uid + 'orientationchange.rad.' + this.uid , self.__setScale );
```
Comment (dylanjha) [inline rad.js:27]: how do these events get fired? `resize.rad`, `orientationchange.rad` I don't follow how this event is being constructed, mind explaining @lisandropuzzolo ?

**Comment by mattnish** [comment]
Hey @lisandropuzzolo would you mind adding documentation to the readme?

Thanks!

**Pull request merged**